src/components/PopupWithConfirm.js

```diff
@@ -16,8 +16,8 @@ export default class PopupWithConfirm extends Popup {
 
   _handleSubmit = (evt) => {
     evt.preventDefault();
-    this._handleConfirmSubmit(this._card, this._cardId);
-    this.closeModal();
+    this._handleConfirmSubmit(this._card, this._cardId)
+    .then(() => {this.closeModal();});
```

Comment:
You need to `catch` possible errors at the very end of any server request.

So, here will be the end of the request. Please, add `catch` here

Reply:
Added